src/components/VContentPage.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="pt-5 md:pt-10" dir="ltr">
+    <div
+      class="px-6 lg:px-0 mb-10 lg:mb-30 md:max-w-4xl xl:max-w-5xl prose prose-sm md:prose-base mx-auto max-w-none prose-headings:font-bold lg:prose-headings:text-3xl prose-h1:text-4xl prose-h1:text-bold lg:prose-h1:text-6xl md:prose-h1:mb-10 prose-h3:text-2xl prose-h3:font-semibold md:prose-h3:mt-10"
```

Comment:
This massive list of `prose-*` classes also scares me. It has all the features of the worst parts of Tailwind IMO (namely, a super long and inscrutable list of class names that are difficult to edit, read, and get a full picture of).

Adding our own `prose` class to `tailwind.css` with these styles using the `@apply` rule would probably be easier to read and maintain :shrug: Just my own feeling about this plugin though.